Makefile

```diff
@@ -0,0 +1,19 @@
+PATH_WITH_TOOLS="`pwd`/node_modules/.bin:${PATH}"
+
+install:
+	npm ci --audit=false --legacy-peer-deps
```

Comment:
Why do we need `--legacy-peer-deps`? Was some dependency chain complaining?

Reply:
i think i added this by mistake - i don't see any issues when i remove this configuration. good catch

Reply:
```suggestion
	npm ci --audit=false
```

Reply:
Nice! Yeah, unfortunately we have it in PRIME becuase storybook's dependency chain is a mess...